recipes/pulp/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set version = "1.6.8" %}
+
+package:
+  name: pulp
+  version: '{{ version }}'
+
+source:
+  sha256: cfecd6a78c2b08539dc29b7e8281f43d729b1089bdb5e491c3c0956aadbb3fc7
+  fn: 'PuLP-{{ version }}.tar.gz'
+  url: https://pypi.io/packages/source/P/PuLP/PuLP-{{ version }}.tar.gz
+
+build:
+  entry_points:
+    - pulptest=pulp:pulpTestAll
+    - pulpdoctest=pulp:pulpDoctest
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pyparsing >=2.0.1
+  run:
+    - python
+    - pyparsing >=2.0.1
+
+test:
+  commands:
+    - pulptest --help
+  imports:
+    - pulp
+    - pulp.solverdir
+
+about:
+  summary: |
+    PuLP is an LP modeler written in python. PuLP can generate MPS or LP files
+    and call GLPK, COIN CLP/CBC, CPLEX, and GUROBI to solve linear problems.
+  home: https://github.com/coin-or/pulp
+  license: MIT
+  doc_url: https://pythonhosted.org/PuLP/
+  dev_url: https://github.com/coin-or/pulp
+  license_family: MIT
+
+extra:
+  recipe-maintainers:
+    - johanneskoester
+    - christopher-schroeder
```

Comment:
@christopher-schroeder, are you happy to [be a conda-forge maintainer](http://conda-forge.readthedocs.io/en/latest/recipe.html#maintainer-role) for the `pulp` package?

Reply:
yes he is. We are sitting in the same office :-)

Reply:
indeed :)